packages/api/src/schema/db-schema.yaml

```
@@ -0,0 +1,336 @@
+      properties:
+        id:
+          readOnly: true
+        kind:
+          readOnly: true
+        userId:
+          readOnly: true
+          index: true
+        createdAt:
+          readOnly: true
+        events:
+          index: true
+          indexType: gin
+        status:
+          readOnly: true
+          properties:
+            lastFailure:
+              readOnly: true
+              properties:
+                timestamp:
+                  readOnly: true
+                error:
+                  readOnly: true
+                response:
+                  readOnly: true
+                statusCode:
+                  readOnly: true
+            lastTriggeredAt:
+              readyOnly: true
+    webhook-response:
+      table: webhook_response
+      properties:
+        id:
+          readOnly: true
+        kind:
+          readOnly: true
+        webhookId:
+          readOnly: true
+          index: true
+        eventId:
+          readOnly: true
+          index: true
+        createdAt:
+          readOnly: true
+    detection-webhook-payload:
+      type: object
+      required:
+        - manifestID
+        - seqNo
+        - sceneClassification
+      properties:
+        manifestID:
+          type: string
+        seqNo:
+          type: number
+        sceneClassification:
+          type: array
+          items:
+            type: object
+            required:
+              - name
+              - probability
+            properties:
+              name:
+                type: string
+              probability:
+                type: number
+    stream:
+      table: stream
+      properties:
+        id:
+          readOnly: true
+        kind:
+          readOnly: true
+        userId:
+          index: true
+        lastSeen:
+          index: true
+        isActive:
+          index: true
+        createdByTokenName:
+          readOnly: true
+        createdByTokenId:
+          readOnly: true
+        createdAt:
+          readOnly: true
+          index: true
+        parentId:
+          index: true
+        streamKey:
+          unique: true
+        playbackId:
+          unique: true
+        wowza:
+          properties:
+            sourceInfo:
+              properties:
+                width:
+                  minValue: 1
+                height:
+                  minValue: 1
+                fps:
+                  minValue: 1
+    multistream-target-patch-payload:
+      table: null
+    session:
+      table: session
+      properties:
+        id:
+          readOnly: true
+        kind:
+          readOnly: true
+        userId:
+          index: true
+        lastSeen:
+          index: true
+        createdAt:
+          readOnly: true
+          index: true
+        parentId:
+          index: true
+        record:
+        recordingStatus:
+          readonly: true
+        recordingUrl:
+          readonly: true
+        mp4Url:
+          readonly: true
```

typo
```suggestion
        recordingStatus:
          readOnly: true
        recordingUrl:
          readOnly: true
        mp4Url:
          readOnly: true
```
(should be moved to `api-schema` anyway, like all other `readOnly`s)